src/DB/Data/ConnectionFactoryInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\DB\Data;
+
+/**
+ * Interface of database connection data factory
+ */
+interface ConnectionFactoryInterface
```

Comment:
What the purpose of this interface if you don't use it anywhere?

Reply:
I have removed the interface for now.